Exercise_1.cpp

```diff
@@ -1,21 +1,64 @@
-#include <stdio.h> 
-  
-// A recursive binary search function. It returns 
-// location of x in given array arr[l..r] is present, 
-// otherwise -1 
-int binarySearch(int arr[], int l, int r, int x) 
-{   
-    //Your Code here 
-} 
-  
-int main(void) 
-{ 
-    int arr[] = { 2, 3, 4, 10, 40 }; 
-    int n = sizeof(arr) / sizeof(arr[0]); 
-    int x = 10; 
-    int result = binarySearch(arr, 0, n - 1, x); 
-    (result == -1) ? printf("Element is not present in array") 
-                   : printf("Element is present at index %d", 
-                            result); 
-    return 0; 
-} 
\ No newline at end of file
+// Time Complexity :O(nlogn)
+// Space Complexity : O(1)

```

Comment:
Just check with the time and space complexities for quick sort. Worst case time is O(n^2) and space is O(logn)
https://www.bigocheatsheet.com/